core-processor/src/lib.rs

```diff
@@ -54,13 +55,15 @@ pub const RE_INIT_STATUS_CODE: StatusCode = 3;
 pub use context::{
     ContextChargedForCode, ContextChargedForInstrumentation, ProcessExecutionContext,
 };
-pub use executor::{calculate_gas_for_code, calculate_gas_for_program, execute_wasm};
+pub use executor::{execute_wasm, PrepareMemoryError};
 pub use ext::{Ext, ProcessorContext, ProcessorError, ProcessorExt};
 pub use handler::handle_journal;
-pub use processor::{
-    precharge_for_code, precharge_for_code_length, precharge_for_instrumentation,
-    precharge_for_memory, precharge_for_program, process,
+pub use precharge::{
```

Comment:
As I know in rust it's more common to use
```rust
mod::function();
```
not like this
```rust
use mod::function;
function();
```

Reply:
Yep, you're right nonetheless this is  a reexport